android/src/main/java/nandorojo/modules/galeria/EdgeToEdgeImageViewerDialogFragment.kt

```diff
@@ -0,0 +1,26 @@
+package nandorojo.modules.galeria
+
+import android.app.Dialog
+import android.os.Bundle
+import androidx.core.view.WindowCompat
+import androidx.core.view.WindowInsetsControllerCompat
+import com.github.iielse.imageviewer.ImageViewerDialogFragment
+
+class EdgeToEdgeImageViewerDialogFragment(private val isAppearanceLightSystemBars: Boolean) :
+    ImageViewerDialogFragment() {
+
+    override fun onCreateDialog(savedInstanceState: Bundle?): Dialog {
+        return Dialog(requireActivity(), R.style.Theme_FullScreenDialog).apply {
+            setCanceledOnTouchOutside(true)
```

Comment:
I kept the `ImageViewerDialogFragment` behavior, just changed the applied theme on the `Dialog` instance.